src/order/atoms.lean

```diff
@@ -270,14 +270,14 @@ instance : is_coatomic α :=
 end is_coatomistic
 end atomistic
 
-/-- A lattice is simple iff it has only two elements, `⊥` and `⊤`. -/
-class is_simple_lattice (α : Type*) [lattice α] [bounded_order α] extends nontrivial α : Prop :=
+/-- An order is simple iff it has only two unique elements, `⊥` and `⊤`. -/
```

Comment:
I'm not sure about the rephrasing, since it might be parsed to imply a simple order can also have "non-unique elements" beyond the two unique ones. (I might also just be too tired to understand English.)
```suggestion
/-- An order is simple iff it has exactly two elements, `⊥` and `⊤`. -/
```